### PR TITLE
Fix 800 - added cancel button in ShowQrActivity

### LINKED ISSUE #800: No cancel button in ShowQrActivity
**Summary:**

The set amount dialog in `ShowQrActivity` only has reset and confirm button there is no option to cancel the dialog

### PR DESCRIPTION
**Pull request by dubeyaayush07**
Fix #800

Screenrecorder-2020-03-09-00-04

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/qr/ui/ShowQrActivity.java [hunk 1]
```diff
@@ -111,6 +111,12 @@ public void onClick(DialogInterface dialog, int which) {
                 showToast("Reset Amount Successful");
             }
         });
+        editTextDialog.setNegativeButton("Cancel", new DialogInterface.OnClickListener() {
```
Comment (devansh-299) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/qr/ui/ShowQrActivity.java]: @dubeyaayush07 please use "Cancel" string from the `strings.xml` file

**Review by devansh-299 — CHANGES_REQUESTED** [review]

**Review by rchtgpt — CHANGES_REQUESTED** [review]
hello @dubeyaayush07, can you please use "Cancel" string from the `strings.xml` file as pointed out by @devansh-299 ?

**Comment by rchtgpt** [comment]
Closing this because of inactivity.

**Comment by dubeyaayush07** [comment]
@rachittt Sorry for the delay I was busy with some other work. If you want I can still make the requested changes

**Comment by rchtgpt** [comment]
@dubeyaayush07 please make the changes.

**Comment by dubeyaayush07** [comment]
@rachittt  done

**Comment by devansh-299** [comment]
> @rachittt done

thanks @dubeyaayush07

**Review by devansh-299 — APPROVED** [review]

**Review by rchtgpt — APPROVED** [review]
thanks @dubeyaayush07

**Pull request merged**